Mathlib/Order/CompleteLattice.lean

```diff
@@ -1293,6 +1293,27 @@ theorem iInf_extend_top {e : ι → β} (he : Injective e) (f : ι → α) :
     ⨅ j, extend e f ⊤ j = iInf f :=
   @iSup_extend_bot αᵒᵈ _ _ _ _ he _
 
+section le
+
+variable {ι : Type*} [PartialOrder ι] (f : ι → α) (i : ι)
+
+theorem biSup_le_eq_sup : (⨆ j ≤ i, f j) = (⨆ j < i, f j) ⊔ f i := by
```

Comment:
I just have one question: you're naming the Data.Set.Lattice lemmas `biUnion_le` etc. - the analogous name here would be `biSup_le`. Is that already taken? (Similarly for the other comments below.)

Reply:
No, but `biInf_le` is `⨅ j ∈ s, f j ≤ f i`.